locales/ru.ts

```diff
@@ -281,6 +281,7 @@ export const ru = {
   },
 
   logOut: {
```

Comment:
`logOut.logOut` is an uninformative property name choice, as is the location in the locale object. The first level of nesting in the locale object contains properties corresponding to the application pages, as well as the `common` property, which contains locale parameters not bound to any particular page. Since the logout modal window is opened from the sidebar, which is common for more parts of the application, it is worth moving the locale description there.
You should also change the property names for better readability:
* `logOut` - describes the title of the modal window
* `reallyWantToLogOut` - content of the modal window.